tests/conftest.py

```diff
@@ -1,61 +1,47 @@
-import json
-import os
 import shutil
 from pathlib import Path
 
 import pytest
 from flywheel_gear_toolkit.utils.zip_tools import unzip_archive
 
-FS_DIR = Path("/usr/local/freesurfer/")
-SUBJECTS_DIR = Path(FS_DIR / "subjects")
+from utils.singularity import run_in_tmp_dir
+
+run_in_tmp_dir()  # run all tests in /tmp/*/flywheel/v0 where * is random
+
+FWV0 = Path.cwd()
 
 
 @pytest.fixture
 def install_gear():
     def _method(zip_name):
         """unarchive initial gear to simulate running inside a real gear.
 
-        This will delete and then install: config.json input/ output/ work/
+        This will delete and then install: config.json input/ output/ work/ freesurfer/
 
         Args:
             zip_name (str): name of zip file that holds simulated gear.
         """
 
-        gear_tests = "/src/tests/data/gear_tests/"
-        gear = "/flywheel/v0/"
-        os.chdir(gear)  # Make sure we're in the right place (gear works in "work/" dir)
+        gear_tests = Path("/src/tests/data/gear_tests/")
+        if not gear_tests.exists():  # fix for running in circleci
+            gear_tests = FWV0 / "tests/data/gear_tests/"
 
         print("\nRemoving previous gear...")
 
-        if Path(gear + "config.json").exists():
-            Path(gear + "config.json").unlink()
+        if Path(FWV0 / "config.json").exists():
+            Path(FWV0 / "config.json").unlink()
 
-        for dir_name in ["input", "output", "work", "freesurfer"]:
-            path = Path(gear + dir_name)
+        for dir_name in ["input", "output", "work", "freesurfer", "templateflow"]:
+            path = Path(FWV0 / dir_name)
```

Comment:
I would've guessed it would be easier to remember which path it was by keeping "dir_"  or "subdir_" in the variable name, rather than just path. This might just be a convention I need to learn, though.

Reply:
That's a good point.  Here it is only used locally so it doesn't really matter what the variable name is.